app/javascript/packages/document-capture/services/upload.js

```diff
@@ -44,7 +63,7 @@ async function upload(payload, { endpoint, csrf }) {
     /** @type {UploadErrorResponse} */
     const errorResult = result;
     const error = new UploadFormEntriesError();
-    error.rawErrors = errorResult.errors;
+    error.rawErrors = errorResult.errors.map(toFormEntryError);
```

Comment:
now that these errors are converted to specific error classes, does `rawErrors` still make sense as a field name?

Reply:
> now that these errors are converted to specific error classes, does `rawErrors` still make sense as a field name?

No, I don't think so. And it exacerbates an existing problem that there's many sorts of "error" objects floating around as it is.

A better name might be `formEntryErrors`.

I don't love that we have to have this container class `UploadFormEntriesError` holding many individual `UploadFormEntryError`, but it seems the best option considering:

- We want to show all errors at once, not just one
- It might technically be possible to throw an array of errors (`throw [ formEntryError, formEntryError ]`), but not only is this uncommon and therefore unexpected, I'd worry that it could interfere with how `Suspense` interprets errors.

Reply:
> A better name might be `formEntryErrors`.

Updated in 2dfb41dd1.